<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1030</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QFrame" name="header_frame">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>300</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QFrame" name="usb_frame">
                 <property name="frameShape">
                  <enum>QFrame::WinPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QHBoxLayout" name="usb_layout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Usb Connection</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="host_frame">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>110</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::WinPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QHBoxLayout" name="qnx_layout">
                    <item>
                     <widget class="QLabel" name="qnx_host">
                      <property name="minimumSize">
                       <size>
                        <width>64</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>QNX IP</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="qnx_ip_label">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::WinPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="text">
                       <string>172.21.0.10</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="linux_layout">
                    <item>
                     <widget class="QLabel" name="linux_host">
                      <property name="minimumSize">
                       <size>
                        <width>64</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Linux IP</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="linux_ip_label">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::WinPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="text">
                       <string>172.21.0.30</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="android_layout">
                    <item>
                     <widget class="QLabel" name="android_host">
                      <property name="minimumSize">
                       <size>
                        <width>64</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Android IP</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="android_ip_label">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>64</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::WinPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="text">
                       <string>172.21.0.40</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <layout class="QHBoxLayout" name="sw_flashing_layout"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>170</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>170</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>Power Source</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="comport_combobox"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="connect_power_source_button">
                      <property name="text">
                       <string>Connect</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QCheckBox" name="channel_1_button">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Channel 1</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="channel_2_button">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Channel 2</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLineEdit" name="voltage_input"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="set_vol_btn">
                      <property name="text">
                       <string>Set</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="power_on_button">
                      <property name="text">
                       <string>Turn ON</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLabel" name="power_source_stats_label">
                    <property name="text">
                     <string>Vout: 12.00 V    Iout: 1.00A</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="tool_area">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>348</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>321</width>
            <height>480</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="load_tools_button">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
          <widget class="QPushButton" name="add_tools_button">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>0</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="log_frame">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QTextEdit" name="log_area"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="prj_reload">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Choose Your Project</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
